addons/l10n_it_edi/models/account_move.py

```diff
@@ -1031,117 +1029,97 @@ def _compose_info_message(self, tree, tags):
     # -------------------------------------------------------------------------
 
     def _l10n_it_edi_export_data_check(self):
-        errors = self._l10n_it_edi_base_export_data_check()
-        if not self._l10n_it_edi_is_simplified():
-            errors += self._l10n_it_edi_export_buyer_data_check()
-        return errors
-
-    def _l10n_it_edi_format_export_data_errors(self):
-        messages = (
-            self._l10n_it_edi_format_errors(move.name + ":" if len(self) > 1 else False, move_warnings)
-            for move in self if (move_warnings := move._l10n_it_edi_export_data_check())
-        )
-        return Markup("<br>").join(messages) or False
+        """ This function checks the Settings, Company, Partners, Moves involved in the
+            sending activity and returns an errors dictionary ready for the
+            actionable_errors widget to display. """
+
+        error_buffer = defaultdict(dict)
+        def save_error(record, error, key):
+            """ Save the error in the error_buffer structure by key """
```

Comment:
What is the advantage of this error buffer?  If there is an error, just add it to the list of errors.  If there are multiple moves, you can do for all at once immediately per error.